src/helpers/helpers.ts

```diff
@@ -130,18 +174,18 @@ export function getImageUrl (filename: string, width?: number): Url {
   return url
 }
 
-export function getEntityIdFromGuid (guid: string): string {
+export function getEntityIdFromGuid (guid: Guid): EntityId {
```

Comment:
I actually prefer string I think typing it to P/Q/L(number) is a bit overkill.

Reply:
(This answers multiple of your code comments)

`Q${number}` extends string so it works wherever string is required but has a bit more information about it.
I don't really see a downside of having it. End users can ignore it and use string too.

For return values at least. Input of functions should probably accept string (or even unknown like I did with some of these functions) as it allows for easier usage.

Also the `something is EntityId` return of TypeScript methods is just typings, the JS code just returns a bool. So not more runtime magic compared to `boolean` or `something is string`.